x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/pagination.tsx

```diff
@@ -19,6 +19,7 @@ export const Pagination: React.FC<{ 'aria-label': string }> = ({ 'aria-label': a
     className="documentsSearchExperience__pagingInfo"
   >
     <EuiFlexItem>
+      {/* @ts-ignore */}
```

Comment:
This is a bug in Search UI types. View passed to Paging component should accept any additional props (for example for data- or area- attributes). Added to issue: https://github.com/elastic/enterprise-search-team/issues/2226